test/parallel/test-dgram-membership.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const multicastAddress = '224.0.0.114';
```

Comment:
I'm on the fence on that. I have a concern that putting that address in `common` will signal to people (like me) who don't fully understand multicast that it's OK for tests in `parallel` and `sequential` to communicate with that address. It's not, I don't think, and that's probably why the existing test that uses that address is in `internet`. This test needs to have an address to test `addMembership()` and `dropMembership()`, which, now that I think about it, I guess means that this test will cause some IGMP traffic on the local network? /cc @nodejs/build in the hopes someone who actually understands how multicast works can say "Yup, this test will send a packet" or "Nope, this test does not send a packet on the network."